moveit_core/trajectory_processing/include/moveit/trajectory_processing/ruckig_traj_smoothing.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2011-2012, Georgia Tech Research Corporation
+ * All rights reserved.
+ *
+ * Author: Tobias Kunz <tobias@gatech.edu>
+ * Date: 05/2012
+ *
+ * Humanoid Robotics Lab      Georgia Institute of Technology
+ * Director: Mike Stilman     http://www.golems.org
+ *
+ * Algorithm details and publications:
+ * http://www.golems.org/node/1570
```

Comment:
This seems strange.

Reply:
How so? I took classes with Mike Stilman back at Georgia Tech and remember that website!

Reply:
I think he was pointing out we copied the license from TOTG but forgot to change it...